digits/tools/create_db.py

```diff
@@ -579,12 +602,15 @@ def _load_thread(load_queue, write_queue, summary_queue,
         if compute_mean:
             image_sum += image
 
-        if backend == 'lmdb':
-            datum = _array_to_datum(image, label, encoding)
-            write_queue.put(datum)
-        else:
-            write_queue.put((image, label))
-
+        try:
+            if backend == 'lmdb':
+                datum = _array_to_datum(image, label, encoding)
+                write_queue.put(datum)
+            else:
+                write_queue.put((image, label))
+        except IOError:  # try to save 16-bit images with PNG/JPG encoding
```

Comment:
Let me catch the error message, too, and set _notification with that error message.